src/plays/react-todo-app/react-todo.css

```diff
@@ -0,0 +1,145 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.play-details-body {
```

Comment:
There is a page-level scroll that appears only for this play, kindly fix that.

![image](https://user-images.githubusercontent.com/3633137/165014559-5741d9c7-1b6f-43ce-a93b-d2a33b480eb8.png)